internal/datasource/host/info_test.go

```diff
@@ -526,11 +526,12 @@ func TestInfo_ContainerInfo(t *testing.T) {
 			info := NewInfo()
 			info.mountInfoLocation = mountInfoFile.Name()
 			info.exec = execMock
+			info.osReleaseLocation = "/non/existent"
```

Comment:
Does hardcoding this here mean that this mergeHostandOSinfo function never gets unit tested? https://github.com/nginx/agent/blob/c25bb76b7f2bfd2959ac67d2b3cc5b6dd2b2727a/internal/datasource/host/info.go#L348 🤔

Reply:
Instead of doing this, is it better if we did:

```
info.osReleaseLocation = osReleaseFile
```

That way we have a mocked OS release file and the code also tests the unexported function? except that `osReleaseFile` also hardcodes stuff so that might be a problem now?

Reply:
> Does hardcoding this here mean that this mergeHostandOSinfo function never gets unit tested?

Hmm, I believe yes. I was thinking that this is already true. If you run this on a Mac then `mergeHostAndOsReleaseInfo` will be skipped since that file doesn't exist.

However, I believe the github pipeline _does_ test this right now: [.github/workflows/ci.yml#L37](https://github.com/nginx/agent/blob/v3/.github/workflows/ci.yml#L37)

I suspect you are correct that the _intent_ was to set `info.osReleaseLocation = osReleaseFile`; however this also fails because the hardcoded data in there is for ubuntu 20.04 and the expectation is ubuntu 22.04.

Reply:
It looks like some of the releasinfo tests were added in https://github.com/nginx/agent/pull/977/files#diff-df29d34cf2163999a2f1dfb2567215ed1d099edf1d93cf4f0afd3a0e809378bc; I suspect the github ci and assertions were already a bit out of sync at that time.

Reply:
We talked this out; the point of this test is to gather container ids correctly not test mergeHostAndOsReleaseInfo.  It would be good to add another test that focuses on that, but it is outside the scope here.

Reply:
so we skip the test then for now?